int32[] tour
float32 deposit
